includes/xml_parsers/EUtilsBioProjectParser.inc

```diff
@@ -160,7 +157,7 @@ class EUtilsBioProjectParser implements EUtilsParserInterface {
 
       //Keys so far are description and action.  No one cares about action?
       if ($key == 'Description') {
-        $org = $child->Organization->Name;
+        $org                  = $child->Organization->Name;
```

Comment:
again... check your whitespace settings.  I dont think this is the drupal standard.
```suggestion
        $org = $child->Organization->Name;
```